docs/en/cookbook/working-with-datetime.rst

```diff
@@ -3,7 +3,7 @@ Working with DateTime Instances
 
 There are many nitty gritty details when working with PHPs DateTime instances. You have to know their inner
 workings pretty well not to make mistakes with date handling. This cookbook entry holds several
-interesting pieces of information on how to work with PHP DateTime instances in Doctrine 2.
+interesting pieces of information on how to work with PHP DateTime instances in ORM.
```

Comment:
```suggestion
interesting pieces of information on how to work with PHP DateTime instances in Doctrine ORM.
```

Reply:
This was intentional. I wanted to reduce "Doctrine ORM" appearing everywhere to prevent it becoming too redundant. I actually thinking about reducing its occurrence in annotations-reference.rst where it is used multiple times.